compaction/compaction.cc

```diff
@@ -451,6 +451,7 @@ class compaction {
     uint64_t _compacting_data_file_size = 0;
     api::timestamp_type _compacting_max_timestamp = api::min_timestamp;
     uint64_t _estimated_partitions = 0;
+    double _estimated_droppable_tombstone_ratio;
     uint64_t _bloom_filter_checks = 0;
```

Comment:
I think we need to initialize _estimated_droppable_tombstone_ratio to 0, to avoid UB.

Reply:
other than that LGTM

Reply:
sorry my bad. fixed it now.

Reply:
no problem :)